### PR TITLE
SAK-50735 Gradebook fix student summary print view

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

**Review by kunaljaykam — CHANGES_REQUESTED** [review]
the css changes are missing

**Comment by ottenhoff** [comment]
> the css changes are missing

I don't know what you're referring to

File: gradebookng/tool/src/webapp/scripts/gradebook-grade-summary.js [hunk 1]
```diff
@@ -268,9 +268,9 @@ GradebookGradeSummary.prototype.setupStudentView = function() {
 GradebookGradeSummary.prototype._print = function(headerHTML, contentHTML, $container) {
   $("#summaryForPrint").remove();
 
-  var $iframe = $("<iframe id='summaryForPrint'>").hide();
+  const $iframe = $("<iframe id='summaryForPrint'>").hide();
   $iframe.one("load", function() {
-    var $head = $iframe.contents().find("head");
+    const $head = $iframe.contents().find("head");
```
Comment (kunaljaykam) [inline gradebookng/tool/src/webapp/scripts/gradebook-grade-summary.js:273]: does replacing `var` with `const` solves the issue?

**Comment by ottenhoff** [comment]
> I tested it locally. I got multiple blank pages with tables but no data. Is that expected?

No, that's the broken behavior.

**Comment by ottenhoff** [comment]
yeah idk what's going on here! closing

**Pull request closed without merging**